### PR TITLE
feat: add sidechains to network toggle

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds sidechains to the network toggle on the top left of the Explorer. This allows users to switch between sidechain networks and other networks on the Explorer more easily.

### Context of Change

Sidechain MVP

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After

Before:
image

After:
(in testnet mode)
image

(in sidechain mode)
image

## Test Plan

CI passes. I tested this in the dev env and made sure everything works as expected.

### CODE REVIEW

File: src/containers/Header/index.js [hunk 1]
```diff
@@ -46,29 +66,104 @@ class Header extends Component {
...
+        onClick={handleClick}
+        onKeyUp={handleClick}
+        tabIndex={0}
+      >
+        <span>{text}</span>
+        {/*
+        Adding this space to provide width between name
+        and selected checkmark
+        */}
+        &nbsp;
```
Comment (khancode) [inline src/containers/Header/index.js:127]: Why not just add css padding between name and checkmark?

**Review by khancode — CHANGES_REQUESTED** [review]

File: src/containers/Header/index.js (hunk 1 above)
Comment (mvadari) [reply src/containers/Header/index.js:127]: @ledhed2222 added this

Comment (ledhed2222) [reply src/containers/Header/index.js:127]: seemed semantic? 🤷

**Review by ledhed2222 — APPROVED** [review]

File: src/containers/Header/index.js (hunk 1 above)
Comment (khancode) [reply src/containers/Header/index.js:127]: Haha ok. I'll approve but we should probably fix this in another PR.

**Pull request merged**